README.md

```diff
@@ -140,17 +140,14 @@ This installs the Cachi2 CLI in [editable mode](https://setuptools.pypa.io/en/la
 which means changes to the source code will reflect in the behavior of the CLI without the need for reinstalling.
 
 You may need to install Python 3.9 in case you want to test your changes against Python 3.9 locally
-before submitting a pull request.
+before submitting a pull request. This is optional though as our gating CI checks compatibility
+with all currently supported Python versions upstream and so we respect your choice to only test
+with your default Python installation. That said, if you test against e.g. Python 3.12 be prepared
+to see potential issues on 3.9 due to new arguments, attributes, operators, etc.
 
-```shell
-dnf install python3.9
-```
-
-The CLI also depends on the following non-Python dependencies:
-
-```shell
-dnf install golang-bin git
-```
+Note though that we also have several non-Pythonic runtime dependencies, so depending on which part
+of cachi2 you want to work on you may need to install further binaries or toolkits. Please refer to
+our Containerfile on what those dependencies are as that always contains the latest information.
```

Comment:
```suggestion
our [Dockerfile](https://github.com/containerbuildsystem/cachi2/blob/main/Dockerfile) on what those dependencies are as that always contains the latest information.
```

Reply:
We should always promote it as a Containerfile, the only reason for having a Dockerfile (and Containerfile as a symlink) is due to Dependabot's limitations, that's not something by choice. It is true that it's currently a symlink, so GH UI will redirect it automatically which may be confusing for some people, but the moment we either replace dependabot or it magically starts supporting some alternative container file names, we don't need to change anything in the docs.

Reply:
I'll update this as suggested.